public/services/IndexService.ts

```diff
@@ -17,6 +17,7 @@ import { IHttpResponse, IHttpService } from "angular";
 import { INDEX } from "../../server/utils/constants";
 import { AddPolicyResponse, GetIndicesResponse, SearchResponse } from "../../server/models/interfaces";
 import { ServerResponse } from "../../server/models/types";
+import { NODE_API } from "../../utils/constants";
```

Comment:
nit: prefer constants under its own folder/module

Reply:
We can discuss outside this PR to see what we will adopt across all plugins and take action after, don't want to spam this PR with moving constants to its own module if we do do that.